core/tests/unit/ldml/ldmlKeyboard.dtd

```diff
@@ -0,0 +1,192 @@
+<!--
```

Comment:
I wonder if we can put this in resources/standards-data/ldml-keyboard along with a README.md detailing where it is sourced from and update strategy?

See for example https://github.com/keymanapp/keyman/blob/d4b8094b137ef2a1aac1eb60a6d0a01658bb525f/resources/standards-data/langtags/readme.md and its corresponding data file.

I realise that this may be problematic/annoying for DOCTYPE decls so feel free to push back on this.

Reply:
I think that's reasonable. Will need a long term strategy for this anyway. Should be versioned.

Reply:
https://github.com/keymanapp/keyman/blob/master/resources/standards-data/unicode-character-database/readme.md this one is not so descriptive :)

Reply:
Personally, I love the simplicity of that readme.